docs/pow.md

```diff
@@ -0,0 +1,71 @@
+# Proof-of-work in the faucet
+
+The faucet uses POW challenges for anti-dos and dynamically shifts the
```

Comment:
This is a public repo and documentation. Don't assume that readers have the same background knowledge on acronyms as you.

```suggestion
The faucet uses POW challenges for anti-Denial-of-Service and dynamically shifts the
```